README.md

```diff
@@ -1,22 +1,32 @@
 # Azure SDK for Rust
 
-This repository is for the development of the [unofficial](https://github.com/Azure/azure-sdk-for-rust/blob/main/FAQ.md#why-is-it-unofficial) Azure SDK for Rust.
+This repository is for the active development of the Azure SDK for Rust. For consumers of the SDK you can follow the links below to visit the documentation you are interested in
 
-## Crates
+* [Overview of Azure SDK for Rust](https://docs.microsoft.com/azure/developer/rust/)
+* [SDK Reference](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk)
+* [Code Samples for Azure SDK for Rust](https://github.com/azure-samples/azure-sdk-for-rust-samples)
+* [Azure REST API Docs](https://docs.microsoft.com/rest/api/)
+* [General Azure Docs](https://docs.microsoft.com/azure)
+* [Share your feedback to our Azure SDK](https://www.surveymonkey.com/r/FWPGFGG)
+
+## Getting Started
+
+To get started with a crate, see the README.md file located in the crate's project folder.  You can find these crate folders grouped by service in the `/sdk` directory.
+
+## Crates available
+
+Each service can have both 'client' and 'management' crates. 'Client' crates are used to consume the service, whereas 'management' crates are used to configure and manage the service.
 
 [All Azure SDK for Rust crates](https://crates.io/teams/github:azure:azure-sdk-publish-rust) are published on crates.io.
 
-### SDK
-These [SDK crates](sdk) are available:
-- [azure_core](https://crates.io/crates/azure_core)
-- [azure_identity](https://crates.io/crates/azure_identity)
-- [azure_data_cosmos](https://crates.io/crates/azure_data_cosmos)
-- [azure_data_tables](https://crates.io/crates/azure_data_tables)
-- [azure_iot_hub](https://crates.io/crates/azure_iot_hub)
-- [azure_security_keyvault](https://crates.io/crates/azure_security_keyvault)
-- [azure_storage_blobs](https://crates.io/crates/azure_storage_blobs)
-- [azure_storage_datalake](https://crates.io/crates/azure_storage_datalake)
-- [azure_storage_queues](https://crates.io/crates/azure_storage_queues)
+### Client modules
+
+Our client crates follow the [Azure Rust SDK guidelines](https://azure.github.io/azure-sdk/rust_introduction.html). These crates allow you to use, consume, and interact with existing resources, for example, uploading a blob. They also share a number of core functionalities including retries, logging, transport protocols, authentication protocols, etc. that can be found in the [azure_identity](https://crates.io/crates/azure_identity) module.
+
+You can find the most up-to-date list of new modules on our [latest page](https://azure.github.io/azure-sdk/releases/latest/index.html#rust).
```

Comment:
Translates better, if ever important:
```suggestion
You can find the most recent list of new crates on our [latest releases page](https://azure.github.io/azure-sdk/releases/latest/index.html#rust).
```

Also, stop calling them "modules". This isn't Go. :smile: